apps/ae_cuckoo_cycle_pow_executables/Makefile

```diff
@@ -0,0 +1,42 @@
+EXECUTABLES = \
+	priv/bin/mean16 \
+	priv/bin/lean16 \
+	priv/bin/verify16 \
+	priv/bin/mean28 \
+	priv/bin/lean28 \
+	priv/bin/verify28 \
+	priv/bin/mean30 \
+	priv/bin/lean30 \
+	priv/bin/verify30
+
+REPO = https://github.com/aeternity/cuckoo.git
+COMMIT = fddbf60
+
+.PHONY: all
+all: \
+	priv/lib/libblake2b.so \
+	$(EXECUTABLES)
+
+.PHONY: clean
+clean:
```

Comment:
What files are there in c_src? It looks strange that clean would delete source code. I would think dist clean would do this bu clean would only delete generated files?
Are there only generated files in c_src?